File: 2026-01/src/day_08/boj_5063/Main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int n;
    cin >> n;

    while (n--) {
        int r, e, c;
        cin >> r >> e >> c;

        if (r > e - c) {
            cout << "do not advertise\n";
        } else if (r < e - c) {
            cout << "advertise\n";
        } else {
            cout << "does not matter\n";
        }
    }
}

File: 2026-01/src/day_08/boj_5063/Main.java
package day_08.boj_5063;

import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer st;

        int n = Integer.parseInt(br.readLine());
        for (int i = 0; i < n; i++) {
            st = new StringTokenizer(br.readLine());
            int r = Integer.parseInt(st.nextToken());
            int e = Integer.parseInt(st.nextToken());
            int c = Integer.parseInt(st.nextToken());

            if (r > e - c) {
                bw.write("do not advertise\n");
            } else if (r < e - c) {
                bw.write("advertise\n");
            } else {
                bw.write("does not matter\n");
            }
        }

        bw.flush();
    }
}
